packages/checkbox/examples/group.example.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
What does this illustrate other than that you can have multiple checkboxes?

Reply:
Not much without the `CheckboxGroup` component! I removed this one.